operator/cmd/mesh/install.go

```diff
@@ -236,9 +240,99 @@ func InstallManifests(iop *v1alpha12.IstioOperator, force bool, dryRun bool, res
 		return iop, err
 	}
 
+	if dryRun {
+		currentObj, err := reconciler.GetObject(iop)
+		if err != nil {
+			return iop, err
+		}
+		obj, err := object.ParseYAMLToK8sObject([]byte(iopStr))
+		if err != nil {
+			return iop, err
+		}
+		expectedObj := obj.UnstructuredObject()
+		result, err := showDifferences(currentObj, expectedObj)
+		if err != nil {
+			return iop, err
+		}
+		fmt.Println(result)
```

Comment:
Not sure what we use in this part of the code, but I think generally we are not doing raw fmt.Printlns and instead have some writer passed it?

Reply:
You can use the `ConsoleLogger `: https://github.com/istio/istio/blob/master/operator/pkg/util/clog/clog.go

Reply:
@morvencao thanks for the pointer. changed it commit #3 as you suggested.